plugins/item_tasks/web_client/views/SelectSingleFileTaskWidget.js

```diff
@@ -59,7 +59,7 @@ var SelectSingleFileTaskWidget = View.extend({
         this.task = params.task;
         this.$('.g-submit-select-task').girderEnable(true);
 
-        this.$('.g-target-result-container').html(selectTaskViewDescriptionTemplate({
+        this.$('.g-target-result-container').html(SelectSingleFileTaskWidgetSelectedTemplate({
```

Comment:
So, now that the `selectSingleFileTaskWidgetSelected.pug` template is so simple (1 templated line), I think you could just inline it with `selectSingleFileTaskWidget.pug`. This `_pickTask` method could just update the selected name by manipulating this.$el('.selected-task-name')`. It's a bit of a violation of the "use templates, not jQuery" rule, but the significantly decreased complexity may be worth it.

This is just a suggestion though, so feel free to keep it as-is if you feel differently.